### PR TITLE
About page and multiple page functionality

### PR DESCRIPTION
**Pull request by baueraidan**
I started working on the about page, at "localhost:3000/about". The index.js file now uses React Router to allow us to have multiple pages. There's also a blank home page, which you can edit in Home.js and Home.css. The navigation bar is now a global component, which we can add to the top of any page using the <NavigationBar /> React component. We might want to create similar React components for things like a page footer.

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -10,6 +10,10 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <!--
+      Use Google Fonts to load "Open Sans", "Roboto", and "Roboto Condensed"
+    -->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans|Roboto+Condensed:300,400|Roboto+Slab" rel="stylesheet">
```
Comment (bkcapretta) [inline frontend/public/index.html]: Let's check in with Ryan about fonts!

**Comment by zallenz** [comment]
gr8

File: frontend/src/About.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                            completion={0.94}
+                            dimension={200}
+                            radius={80}
+                            label="of students who did not buy textbooks said they were concerned this would hurt their grade"
+                        />
+                    </div>
+                </div>
+                <div className="SectionWrapper" style={{backgroundColor: '#F5A23A'}}>
+                    <div className="SectionTitle">Our Solution</div>
+                    <div className="SectionSubtitle">We're making textbooks more affordable.</div>
```
Comment (bkcapretta) [inline frontend/src/About.js]: Using the apostrophe is making it weird, so use &#39; instead to act as an apostrophe

**Comment by bkcapretta** [comment]
Make sure to fix merge conflicts before pushing to master, otherwise looks great!

**Pull request merged**